src/io/flutter/view/FlutterView.java

```diff
@@ -373,6 +401,37 @@ private void debugActiveHelper(FlutterApp app, @Nullable InspectorService inspec
 
     toolWindow.setIcon(ExecutionUtil.getLiveIndicator(FlutterIcons.Flutter_13));
 
+    if (JxBrowserManager.getInstance().isInstalled()) {
+      final DevToolsManager devToolsManager = DevToolsManager.getInstance(app.getProject());
+
+      if (devToolsManager.hasInstalledDevTools()) {
+        addBrowserInspectorViewContent(app, inspectorService, toolWindow);
+      }
+      else {
+        final CompletableFuture<Boolean> result = devToolsManager.installDevTools();
+        result.thenAccept(o -> addBrowserInspectorViewContent(app, inspectorService, toolWindow));
```

Comment:
do we need to handle if devtools failed to install?

Reply:
Yes, what would make sense here? I could pull the code below that starts the old inspector into a separate function and run that instead. Or maybe just display a message?